### DISCUSSION TITLE
DAI 2024-2025 - Practical work 1 - Thomas Stäheli and Thirusan Rajadurai

### DISCUSSION THREAD

**Discussion by thomasstaheli** [heig-vd-dai-course/heig-vd-dai-course#329] [Show and tell]
Our goal is to make a simple CLI application to manage .bmp file in order to add diffrentes types of filters, for example add a gray filter, add a blur effect, modify all the pixels images with certains colors mentionned in the command (like only black and white pixel).
We have an exaustive list of what we went to implement, but maybe we will not have time to implement all of them.

**Comment by GeraudSilvestri** [comment]
Is good to me. I warn you that you have limited time, so restrict to a few filters at first. Once those work and are clean, add new ones.

**Reply by thomasstaheli** [reply]
Okay, thank you👍

**Comment by thomasstaheli** [comment]
Good evening, 

As asked, this is the link to our repository : [Repository CLI Manager BMP](https://github.com/thomasstaheli/DAI_labo1_CLI_Image_Manager.git)
And this is the commit hash for the evaluation :  6355b236ab4e10e7b0693ce64a759aedf351a0d0 [Link to hash](https://github.com/thomasstaheli/DAI_labo1_CLI_Image_Manager/commit/6355b236ab4e10e7b0693ce64a759aedf351a0d0)

See you tomorow ! 👍

**Reply by ludelafo** [reply]
Hello @thomasstaheli, @Thiru-sama,

Could you add @hadrylouis to your repository so he can grade your work?

Thank you!

**Reply by ludelafo** [reply]
And me as well, I didn't accept the invitation in time.

**Reply by thomasstaheli** [reply]
Yes, it's done ! 👍

**Reply by ludelafo** [reply]
Thank you very much!
